### PR TITLE
Extract test cases and test case descriptions to split testing of svgOMG...

### PR DESCRIPTION
**Pull request by dirkschulze**
... and svgWrite in a next step.

### CODE REVIEW

**Comment by dirkschulze** [comment]
Holding this back for now. @jhatwich could you tell me why this lowers the test coverage?

File: tests/svgWriter.test.js [hunk 1]
```diff
@@ -227,182 +228,16 @@ describe('svgWriter', function (){
...
-        it("should support gradient overlay in combination with color overlay", function () {
-            compareResults('gradient-color-overlay');
-            
-            _isLastTest = true;
-        });
 
+        for (var i = 0, end = database.length; i < end; i++) {
+            var test = database[i].test;
+                desc = database[i].desc;
+            it(desc, function () {
```
Comment (jhatwich) [inline tests/svgWriter.test.js]: the variable 'test' is not stable and will change before the test closure executes.  using this loop each test will execute the comparison of the last test in the list.

define a function that takes test and desc variables and call "it(..." from within that function closure to make the test and desc references stable

**Comment by jhatwich** [comment]
I think the data-structure should also support it.skip so we can temporarily disable borked tests while things are patched

**Comment by dirkschulze** [comment]
This is now covered by pr147 including it.skip

**Pull request closed without merging**